apps/swirl-docs/src/scripts/build-search.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  DOCUMENTATION_CATEGORY,
+  createStaticPathsData,
+  DocumentationCategory,
+  StaticPathMap,
+} from "@swirl/lib/docs";
+import { generateSerializableDocumentation } from "@swirl/lib/docs/src/singleDoc";
+import { ALGOLIA_INDEX } from "@swirl/lib/search";
+import algoliasearch from "algoliasearch";
+
+import dotenv from "dotenv";
```

Comment:
**suggestion:** Could you add a `.env.sample` file to the repo, including all used keys with empty values? Makes it easier to set it up locally. Could also be part of the setup steps for local development in the README.